ui/material.py

```diff
@@ -107,6 +107,27 @@ def draw(self, context):
                 op.preset = preset
 
 
+class LUXCORE_PT_settings(MaterialButtonsPanel, Panel):
+    bl_label = "Settings"
+    bl_context = "material"
+    bl_options = {'DEFAULT_CLOSED'}
+
+    @classmethod
+    def poll(cls, context):
+        engine = context.scene.render.engine
+        return engine == "LUXCORE"
+
+    def draw(self, context):
+        layout = self.layout
+
+        mat = context.material
+        cmat = mat.cycles
```

Comment:
```suggestion
```
This is not used, but causes errors when the panel is opened while an object without material is selected:
```
Traceback (most recent call last):
  File "/home/simon/.config/blender/2.79/scripts/addons/BlendLuxCore/ui/material.py", line 124, in draw
    cmat = mat.cycles
AttributeError: 'NoneType' object has no attribute 'cycles'
```